src/httpserver/file_io/file_input.clj

```diff
@@ -0,0 +1,17 @@
+(ns httpserver.file-io.file-input
+  (:require [clojure.java.io :as io])
+  (:import java.io.FileInputStream))
+
+(defn- partial-content [array-of-bytes start end]
+  (subvec (mapv byte array-of-bytes) start end))
+
```

Comment:
As I have been working on the partial content handler. I realize this should probably not be in the `file-input` ns and instead should be in the `partial-content` handler ns.